meshparty/mesh_skel_utils.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from scipy import sparse
+import trimesh
+from . import utils
+
+
+#function to find the path along the mesh between a point on the mesh and the skeleton (closest skeleton point)
+#if the point is not on the mesh, the point used is the mesh vertex that is closest to the point
+
+def point_to_skel_meshpath(mesh,sk,pt):
+
```

Comment:
doc string please!

Reply:
doc string added and tests added.

Reply:
Can you follow the google style i'm replicating in the docs branch?  Parameters with underscores etc

Reply:
Oui! Done!